website/docs/r/labels.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "helper"
```

Comment:
Not really sure how to categorize this, maybe it doesn't need a category at all.

Reply:
I think we can omit the subcategory here and let it leave under `Resources` together with `kubernetes_manifest`.